engine/lite/request-response/runtime/src/test/scala/pl/touk/nussknacker/engine/requestresponse/deployment/FileProcessRepositoryTest.scala

```diff
@@ -50,6 +51,6 @@ class FileProcessRepositoryTest extends FunSuite with Matchers {
     deployment.processVersion.modelVersion shouldBe Some(3)
     deployment.processVersion.user shouldBe "testUser"
     deployment.deploymentTime shouldBe 5
-    deployment.graphProcess shouldBe GraphProcess(graphProcessJson)
+    deployment.graphProcess shouldBe ProcessMarshaller.fromJsonUnsafe(canonicalProcessJson)
```

Comment:
Is is back compability?

Reply:
it is backward compatible on level of json but not on level of programmable api. Level of json is more important here. Also this code will vanish as I mentioned above